app/models/reports/export_raw.rb

```diff
@@ -84,6 +84,8 @@ def default_report_hash
         journal_date: ->(od) { od.journal.journal_date if od.journal },
         reconciled_note: :reconciled_note,
         reconciled_at: :reconciled_at,
+        invoice_number: ->(od) { od.statement.try(:invoice_number) },
+        charge_for: ->(od) { od.price_policy.try(:charge_for) },
```

Comment:
@jhanggi -- this field is nullable. It's a little unclear to me from the ticket--are they okay with this being null? I'm not sure it would ever be set to "quantity", for instance...